### DISCUSSION TITLE
Can public pages be indexed?

### DISCUSSION THREAD

**Discussion by scouech** [outline/outline#1928] [Self Hosting] [answered]
hey!

Looking into using outline for our team. I'm wondering if pages shared publicly can be indexed by google etc? 
We would love to be able to have some content public, and that people could find it on google for example. Is it possible?

Cheers!

**Comment by tommoor** [comment, marked as answer]
Currently the entire application is disallowed to search crawlers – we er'd on the side of caution here as it's entirely possible to accidentally share private documents and you wouldn't want them scooped up by crawlers.

It's interesting to think how we could make this opt-out though, plans to share an entire document tree or a collection are on the roadmap – I think this could be a natural place to add the option of "Make indexable"

**Reply by scouech** [reply]
thanks for your reply!
yeah totally! I see the potential for public FAQs etc. Or using Outline a bit like a public "wiki" sort of things :)

**Reply by shtefcs** [reply]
> Currently the entire application is disallowed to search crawlers – we er'd on the side of caution here as it's entirely possible to accidentally share private documents and you wouldn't want them scooped up by crawlers.
> 
> It's interesting to think how we could make this opt-out though, plans to share an entire document tree or a collection are on the roadmap – I think this could be a natural place to add the option of "Make indexable"

Hey, is this already added?
